src/Sarif.Driver.UnitTests/Sdk/SkimmerBaseTestResources.Designer.cs

```diff
@@ -8,7 +8,7 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace Microsoft.CodeAnalysis.Sarif.Driver.Sdk {
+namespace Microsoft.CodeAnalysis.Sarif.Driver.UnitTests.Sdk {
```

Comment:
>Microsoft.CodeAnalysis.Sarif.Driver.UnitTests.Sdk [](start = 10, length = 49)

why have you renamed these namespaces? generally, the unit test namespaces match the assembly under test.

Reply:
I don't recall renaming this manually. I'll investigate.